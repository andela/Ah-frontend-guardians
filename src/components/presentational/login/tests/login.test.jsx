import React from 'react';
import { shallow } from 'enzyme';
import Form from '../login';

const mockedProps = {
    errors:{
        errors: {
            error: 'error'
        }
    }
}
describe(`Login`, () => {
    let wrapper;
    const mockLoginfn = jest.fn()

    describe('Login', () => {
        beforeEach(() => {
            wrapper = shallow(<Form {...mockedProps} />)
        })

        it("Login form should render", () => {
            expect(wrapper).toMatchSnapshot();
          });

        it('should render a form', () => {
            expect(wrapper.find('form').length).toEqual(1);
        })
        it('should render an input', () => {
            expect(wrapper.find('input').length).toEqual(2);
        })
        it('should render a paragraph', () => {
            expect(wrapper.find('p').length).toEqual(3);
        })
    })
    });

