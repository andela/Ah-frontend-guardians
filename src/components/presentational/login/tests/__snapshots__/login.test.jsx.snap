// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Login Login form should render 1`] = `ShallowWrapper {}`;
